src/Stratis.Bitcoin/Base/BaseFeature.cs

```diff
@@ -278,25 +276,34 @@ public override void Dispose()
             this.logger.LogInformation("Flushing peers...");
             this.flushAddressManagerLoop.Dispose();
 
+            this.logger.LogInformation("Disposing peer address manager...");
             this.peerAddressManager.Dispose();
 
-            this.partialValidator.Dispose();
```

Comment:
what is the rationale here?

Reply:
@dangershony wanted this object to be disposed of in ConsensusManager, not base feature.

Also we noticed this object wasn't used in base feature.